tools/install_local_wheel.py

```diff
@@ -27,7 +27,7 @@ def configure_logging(lvl=logging.WARNING, logfile=None):
 def main():
     # Setup the required arguments for this script
     usage = """
-Install the wheel found in bin/<driver>/dist
+Install the wheel found in generated/<driver>/dist
```

Comment:
I grepped for "bin" and found two similar places where updates might be required:

- https://github.com/ni/nimi-python/blob/969ceaa214392e68850d841d1b2599cb50ffde74/tools/build_release.py#L137
- https://github.com/ni/nimi-python/blob/969ceaa214392e68850d841d1b2599cb50ffde74/tools/simple_mako.py#L78

Reply:
First one is changed to generated
Second the description was fixed - it was wrong before.